weed/pb/grpc_client_server.go

```diff
@@ -144,6 +144,12 @@ func WithGrpcClient(streamingMode bool, fn func(*grpc.ClientConn) error, address
 		defer grpcConnection.Close()
```

Comment:
The connection is closed here. The extra close is not needed?

Reply:
This is the best idea I've come up with so far - this is to make sure there are no errors when closing the connection.

Reply:
right. but closing twice is not necessary.

Reply:
> right. but closing twice is not necessary.

Now I'm more inclined to the fact that the firewall does not allow to keep very long sessions and closes it on its side, but I still don't know how to recreate the session on the application side

Reply:
there used to be a 10 hour connection time limit. May need to add it back.

Reply:
reverted by 41dfe27102b16ae1fd9cc6f17243d2599295f297